### ISSUE TITLE
Confusing duplicate varying preferential TFA-303221 vs. LaCrosse-TX141THBv2 decoder

### ISSUE THREAD

**Issue by DigiH** [NorthernMan54/rtl_433_ESP#26]
While I'm aware of the duplicate/merge comment in the [tfa_30_3221](https://github.com/NorthernMan54/rtl_433_ESP/blob/master/src/rtl_433/devices/tfa_30_3221.c) decoder

`This is the same as LaCrosse-TX141THBv2 and should be merged.`

and this being taken over from the current [rtl_433 decoders](https://github.com/merbanan/rtl_433/tree/master/src/devices)

It is very confusing when using the decoders with OMG, where after a restart of OMG either the [TFA-303221](https://github.com/NorthernMan54/rtl_433_ESP/blob/master/src/rtl_433/devices/tfa_30_3221.c) or the [LaCrosse-TX141THBv2](https://github.com/NorthernMan54/rtl_433_ESP/blob/master/src/rtl_433/devices/lacrosse_tx141x.c) seems to get preferential treatment in the creation of MQTT messages, while stil both do show up over time.

Since the ID stays the same, but it changing with battery changes of the sensors, I've been using the channel to differentiate my fridge/freezer sensors. This channel is however different with the two decoders.

I only came across this when realising that my current TFA-303221 decoder recognition didn't get the regular updates it used to, and found that currently the LaCrosse-TX141THBv2 decoder is the preferred one, producing more regular MQTT messages. 

I'm not sure if the same behaviour is happening with rtl_433 on a different platform, e.g. Raspberry Pi, or if there, both decoders are recognised simultaneously, producing duplicate MQTT messages, but with OMG and rtl_433_ESP there is a varying preferential decoder result.

Here are the duplicates, after waiting a while to get proper readings from both decoders.

```
{"model":"LaCrosse-TX141THBv2","id":175,"channel":1,"battery_ok":1,"temperature_C":4,"humidity":71,"test":"No","mic":"CRC","protocol":"LaCrosse TX141-Bv2, TX141TH-Bv2, TX141-Bv3, TX141W, TX145wsdth sensor","rssi":-38,"duration":240242}
{"model":"TFA-303221","id":175,"channel":2,"battery_ok":1,"temperature_C":4,"humidity":71,"sendmode":0,"mic":"CRC","protocol":"TFA Dostmann 30.3221.02 T/H Outdoor Sensor","rssi":-39,"duration":242160}

{"model":"LaCrosse-TX141THBv2","id":54,"channel":0,"battery_ok":1,"temperature_C":-21.2,"humidity":78,"test":"No","mic":"CRC","protocol":"LaCrosse TX141-Bv2, TX141TH-Bv2, TX141-Bv3, TX141W, TX145wsdth sensor","rssi":-52,"duration":259944}
{"model":"TFA-303221","id":54,"channel":1,"battery_ok":1,"temperature_C":-21.2,"humidity":78,"sendmode":0,"mic":"CRC","protocol":"TFA Dostmann 30.3221.02 T/H Outdoor Sensor","rssi":-43,"duration":236051}
```

I have forked and created my own MY_DEVICES list to remedy this for my setup for now, and ultimately this is a rtl_433 issue, but I wanted to point out this varying preferential decoder recognition, in case others have similar issues.

P.S.: Or since rtl_433_ESP only uses a subset of the rtl_433 decoders anyway, the TFA-303221 could be excluded from the list to avoid this ;)

https://github.com/merbanan/rtl_433/issues/1652

**Comment by DigiH** [comment]
After reading through the comments in the rtl_433 issues, with the proposed **removal of the TFA-303221 protocol** I reverted my MY_DEVICES solution and am now running with the changes suggested in the above PR and the relevant openHAB changes for the new model recognition.

**Comment by DigiH** [comment]
Closing as fixed with 0.1.4 and https://github.com/NorthernMan54/rtl_433_ESP/pull/43 there are only very sporadic TFA-303221 recognitions of my LaCrosse-TX141THBv2 thermometers.

Thanks
